lib/omnifocus.rb

```diff
@@ -123,7 +125,7 @@ def prepopulate_existing_tasks
     prefixen = self.class._plugins.map { |klass| klass::PREFIX rescue nil }
     of_tasks = nil
 
-    prefix_re = /^(#{Regexp.union prefixen}(?:-[\w\s.-]+)?\#\d+)/
+    prefix_re = /^(#{Regexp.union prefixen}(?:-[\w\s\p{Han}.-]+)?\#\d+)/
```

Comment:
Can you tell me if `\p{L}` covers over `\p{Han}`?

Reply:
Hello?

Reply:
@tenderlove any help here?

Reply:
Have test \p{L} can replace \p{Han}.